mongo/datadog_checks/mongo/mongo.py

```diff
@@ -215,7 +221,7 @@ def _check(self):
             dbnames = api.list_database_names()
             self.gauge('mongodb.dbs', len(dbnames), tags=tags)
 
-        self.refresh_collectors(api.deployment_type, mongo_version, dbnames, tags)
+        self.refresh_collectors(deployment, mongo_version, dbnames, tags)
```

Comment:
Why does `api.deployment_type` need to be changed here?

Reply:
It doesn't need to be changed, but I just noticed we weren't utilizing the variable here: https://github.com/DataDog/integrations-core/pull/10843/files#diff-3eb8a896c42d86cc06866848b122aba1329c2fd1feffdf9795c14bb434699286R205